<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchRoomPage</class>
 <widget class="QWidget" name="SearchRoomPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>开始日期</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DateTimeEdit" name="CB_DATE_START"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>结束日期</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DateTimeEdit" name="CB_DATE_END"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="ROOM_NUM_SEARCH">
       <property name="title">
        <string>按照房间号查询</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>房间号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_ROOM_NUMBER"/>
        </item>
        <item>
         <widget class="QPushButton" name="PB_SEARCH_NUMBER">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
       <item>
        <widget class="QGroupBox" name="horizontalGroupBox">
         <property name="title">
          <string>房间状态信息</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="RoomSummaryTable" name="TB_SUMMARY"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="horizontalGroupBox">
         <property name="title">
          <string>空闲房号</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QListWidget" name="LST_ROOM_REMAIN"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RoomSummaryTable</class>
   <extends>QTableWidget</extends>
   <header>Widgets/RoomSummaryTable.h</header>
  </customwidget>
  <customwidget>
   <class>DateTimeEdit</class>
   <extends>QComboBox</extends>
   <header>Widgets/DateTimeEdit/date_time_edit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
